src/packages.jl

```diff
@@ -123,19 +121,19 @@ end
 # thus nec. to start Julia anew
 
 """
-    remove(spec::String; quiet::Bool = false)
+    remove(spec::String; interactive::Bool = true, pkgdir::AbstractString = DEFAULT_PKGDIR, quiet::Bool = false)
```

Comment:
Perhaps write `GAP.Packages.DEFAULT_PKGDIR`, since the variable is not exported?